src/main/java/org/graylog/integrations/notifications/types/SlackClient.java

```diff
@@ -52,16 +56,17 @@ public SlackClient(OkHttpClient httpClient) {
      * @throws TemporaryEventNotificationException - thrown for network or timeout type issues
      * @throws PermanentEventNotificationException - thrown with bad webhook url, authentication error type issues
      */
-    public void send(SlackMessage message, String webhookUrl) throws TemporaryEventNotificationException, PermanentEventNotificationException {
+    public void send(SlackMessage message, String webhookUrl) throws TemporaryEventNotificationException,
+            PermanentEventNotificationException, JsonProcessingException {
 
         final Request request = new Request.Builder()
                 .url(webhookUrl)
-                .post(RequestBody.create(MediaType.parse(APPLICATION_JSON), message.getJsonString()))
+                .post(RequestBody.create(MediaType.parse(APPLICATION_JSON), objectMapper.writeValueAsString(message)))
                 .build();
 
-        LOG.debug("Posting to webhook url <{}> the paylod is <{}>",
+        LOG.debug("Posting to webhook url <{}> the payload is <{}>",
                 webhookUrl,
-                message.getJsonString());
+                "");
```

Comment:
Was this supposed to be `objectMapper.writeValueAsString(message` ?

Reply:
yup, good catch. I just blew away the original parameter to get things to compile originally